examples/rivets/index.html

```diff
@@ -0,0 +1,123 @@
+<!doctype html>
+<html lang="en" data-framework="rivets">
+	<head>
+		<meta charset="utf-8">
+		<title>Rivets • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+	</head>
+	<body>
+		<script type="text/template" id="todos">
+			<header id="header">
```

Comment:
yep I was thinking about the work last night required for a dynamic suite, should actually be quite simple :cocktail: